host_cleanup.sh

```diff
@@ -41,25 +41,39 @@ else
   sudo systemctl restart NetworkManager
 fi
 
+# handle upgrade from legacy network scripts
+if [ -e /etc/sysconfig/network-scripts/ifcfg-${PROVISIONING_NETWORK_NAME} ]; then
+    sudo rm -f /etc/sysconfig/network-scripts/ifcfg-${PROVISIONING_NETWORK_NAME}
+fi
+if [ -e /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME} ]; then
+    sudo rm -f /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME}
+fi
+if [ -e /etc/sysconfig/network-scripts/ifcfg-${PRO_IF} ]; then
+    sudo rm -f /etc/sysconfig/network-scripts/ifcfg-${PRO_IF}
+fi
+if [ -e /etc/sysconfig/network-scripts/ifcfg-${INT_IF} ]; then
+    sudo rm -f /etc/sysconfig/network-scripts/ifcfg-${INT_IF}
+fi
+
```

Comment:
If you add `systemctl restart network.service || true` here you will allow to keep using master branch on Stream 8 bootstrapped with very old dev-scripts

Reply:
we can add this in a follow up as part of cleaning

Reply:
#1637